lib/toxiproxy.rb

```diff
@@ -229,7 +229,9 @@ def self.ensure_webmock_whitelists_toxiproxy
     endpoint = "#{uri.host}:#{uri.port}"
     WebMock::Config.instance.allow ||= []
     unless WebMock::Config.instance.allow.include?(endpoint)
-      WebMock::Config.instance.allow << endpoint
+      WebMock.disable_net_connect!(
```

Comment:
I think this semantic is very different from before no? This will actually disable the connection right away, with the updated list. Before the intent was to only add it to the list of whitelisted endpoints, not actually disabling the connection.

I'm not sure I understand the problem 😓 Also this method seems to be deprecated too https://github.com/bblimke/webmock/blob/e386be1cb3652c83d76fec57847ea4bed581b992/lib/webmock/webmock.rb#L156.